README.md

```diff
@@ -32,8 +32,8 @@ OPTIONS:
 
 SUBCOMMANDS:
     new                  Setup and create a new smart contract project
-    build                Compiles the smart contract
-    generate-metadata    Generate contract metadata artifacts
+    build                Compiles the contract, generates metadata, bundles both together in a '.contract' file
+    check                Check that the Wasm builds; does not optimize, generate metadata, or bundle
```

Comment:
```suggestion
    check                Check that the code builds as wasm; does not output any build artifact to the top level `target/` directory.
```

This statement is of course only true if we prevent copying stuff to the top level `target/` dir on "check".